scripts/EGAP/EGAP_tests.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+from create_EGAP_json import (schema_to_spreadsheet_mapping,
+	make_project_dict,
+	make_registration_dict,
+	other_mapping,
+)
+
+HEADER_ROW = ['POST DATE',
+	'ID',
+	'STATUS',
+	'TITLE',
+	'B2 AUTHORS',
+	'EMAIL',
+	'B3 ACKNOWLEDGEMENTS',
+	'B4 FACULTY MEMBER?',
+	'B5 PROSPECTIVE OR RETROSPECTIVE?',
+	'B6 EXPERIMENTAL STUDY?',
+	'B7 DATE OF START OF STUDY',
+	'B8 GATE DATE',
+	'B8 FORMERLY GATED UNTIL',
+	'B9 PRESENTED AT EGAP MEETING?',
+	'B10 PRE-ANALYSIS PLAN WITH REGISTRATION?',
+	'C1 BACKGROUND',
+	'C2 HYPOTHESES',
+	'C3 TESTING PLAN',
+	'C4 COUNTRY',
+	'C5 SAMPLE SIZE',
+	'C6 POWER ANALYSIS?',
+	'C7 IRB APPROVAL?',
+	'C8 IRB NUMBER',
+	'C9 DATE OF IRB APPROVAL',
+	'C10 INTERVENTION IMPLEMENTER',
+	'C11 REMUNERATION?',
+	'C12 PUBLICATION AGREEMENT?',
+	'C13 JEL CODES',
+	'METHODOLOGY',
+	'POLICY']
+
+TEST_ROW_WITH_OTHER = ['03/05/2017 - 17:00',
+	'20170305AA',
+	'Status is not saved, so this field doesnt matter',
+	'The members of Nsync',
+	'Justin Timberlake, Joey Fatone, Lance Bass',
+	'doesnt@matter.com',
+	'We acknolowledge Chris Kirkpatrick',
+	'Justin Timberlake is a faculty Member',
+	'This is my other response for prospective',
+	'Yes',
+	'05/01/2017',
+	'05/01/2020',
+	'',
+	'No',
+	'No',
+	'Test background',
+	'test hypothesis',
+	'This is my testing plan',
+	'Switzerland',
+	'3242',
+	'This is a power analysis other response',
+	'This is an other irb response',
+	'343434',
+	'03/06/2017',
+	'This is an other intervention response',
+	'This is an other renumeration response',
+	'This is an other publication agreement response',
+	'Jel Code',
+	'Survey Methodology',
+	'Gender']
+
+TEST_ROW_WITH_OTHER_AUTHORS = [
+ 	{'name': 'Justin Timberlake', 'email': 'jt@gmail.com'},
+ 	{'name': 'Joey Fatone'},
+ 	{'name': 'Lance Bass', 'email': 'lBass@gmail.com'}]
+
+TEST_ROW = ['05/05/2018 - 17:00',
+	'20180505AA',
+	'Status is not saved, so this field doesnt matter',
+	'The members of Backstreet boys',
+	'Nick Carter, Brian Littrell, Ph.D., AJ McLean',
```

Comment:
Would it be worth adding 'U.S. Department of State, Bureau of Conflict and Stabilization Operations (CSO) ' or similar to this test?

Reply:
Added!